sharktank/sharktank/ops/quantized_impls.py

```diff
@@ -96,6 +111,180 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def verify_quantized_shape(actual: tuple[int, ...], expected: tuple[int, ...]):
```

Comment:
I'm not sure if this is too cursed, but we could move this into `_quantize_trampoline` and then you don't need to call it at the end of quanizer.